autoload/airline/extensions/branch.vim

```diff
@@ -256,6 +259,13 @@ function! airline#extensions#branch#head()
     endif
   endif
 
+  if empty(heads)
+    if s:has_custom_scm
+      let Fn = function(g:airline#extensions#branch#custom_head)
+      let b:airline_head = Fn()
```

Comment:
wrap it in a try/catch please, just in case the is an error

Reply:
Done.